notebooks/Emukit-basic-use-of-the-library.ipynb

```diff
@@ -0,0 +1,349 @@
+   "source": [
+    "# Basic use of the Emukit\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# General imports and parameters of figures should be loaded at the beginning of the overview\n",
+    "\n",
+    "### General imports\n",
+    "%matplotlib inline\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import colors as mcolors\n",
+    "\n",
+    "### --- Figure config\n",
+    "colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)\n",
+    "LEGEND_SIZE = 15\n",
+    "TITLE_SIZE = 25\n",
+    "AXIS_SIZE = 15"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to illustrate how the a model can wrapped and used in different tasks in Emukit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Navigation\n",
+    "\n",
+    "1. [Load your objective, collect some data, build a model](#1.-Load your objective, collect some data, build a model)\n",
+    "\n",
+    "2. [Load the elements to solve your problem and run the decision loop if needed](#2.-Load the elements to solve your problem and run the decision loop if needed)\n",
+    "\n",
+    "3. [Conclusions](#3.-Conclusions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.  Load your objective, collect some data, build a model\n",
+    "\n",
+    "These steps are common to all methods in Emukit.Here we illustrate how to do it with the branin function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from emukit.test_functions import branin_function\n",
+    "from emukit.core import ParameterSpace, ContinuousParameter\n",
+    "from emukit.experimental_design.model_free.random_design import RandomDesign\n",
+    "from GPy.models import GPRegression\n",
+    "from emukit.model_wrappers import GPyModelWrapper"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define the objective function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 87,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f, _ = branin_function()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define the parameter space"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 88,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_space = ParameterSpace([ContinuousParameter('x1', -5, 10), ContinuousParameter('x2', 0, 15)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Collect some points and build a model of f"
```

Comment:
```suggestion
    "#### Collect some observations of f"
```
You don't fit a model in this cell and the next cell has a description about fitting the model

Reply:
Done